src/enums/filingSubTypes.ts

```diff
@@ -1,16 +1,21 @@
+import { AmalgamationTypes, RestorationTypes } from '@bcrs-shared-components/enums'
+
 /** Filing sub-types used by Legal API. */
 // FUTURE: move these to shared enum and import that instead
 export enum FilingSubTypes {
   ADMIN_FREEZE = 'adminFreeze',
   ADMIN_UNFREEZE = 'adminUnfreeze',
+  AMALGAMATION_REGULAR = AmalgamationTypes.REGULAR,
```

Comment:
Interesting that SC isn't complaining about this (A mixture of enum values + strings). In AUTH WEB, it didn't like that.

Reply:
🤷‍♂️ 

Anyway, this _should_ be allowed, because all the values are strings.

Reply:
Agreed.